### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by lukeh990** [stalwartlabs/smtp-server#9] [Q&A] [answered]
I've followed the steps in the getting started guide, and I've got the SMTP up and running and using LMTP to connect to a Stalwart JMAP server. But I don't know how to actually start sending and receiving mail. Is there a guide I am missing? Any help is appreciated.

**Comment by mdecimus** [comment, marked as answer]
In order to send and receive emails you need a JMAP client, you can check [this list](https://jmap.io/software.html) for alternatives. However, if you would like to use a standard email client such as Outlook or Thunderbird then you'll need to install [Stalwart IMAP](https://github.com/stalwartlabs/imap-server/) which proxies IMAP requests and converts them into JMAP requests.

**Reply by lukeh990** [reply]
I started to figure some of that out. But it's still helpful. Just one more question, what do I need to do on the DNS side? I have an A record for jmap.domain.com and mail.domain.com (For SMTP server). Do I need an MX record or something?
